### ISSUE TITLE
Spelling mistake "staring" should be "starting"

### ISSUE THREAD

**Issue by MikeTheCanuck** [rusq/slackdump#291]
**Describe the bug**
When the Export starts, the console output says "staring" where I'm guessing it means to say "starting"
```
> checking user cache...
2024/05/20 09:43:15   caching users for the first time
2024/05/20 10:21:09 Export:  staring export to: <zip archive: export.zip>
```

**Authentication type (please read)**
n/a

**To Reproduce**
1. Download the macOS binary
2. Setup auth using .env file
3. Run slackdump with this command: `./slackdump -export export.zip -export-type mattermost -download -w WORKSPACE_NAME CHANNEL_ID`

**Expected behavior**
`Export:  starting export to: <zip archive: export.zip>`

**Output**
`Export:  staring export to: <zip archive: export.zip>`

**Desktop (please complete the following information):**
 - OS: macOS
 - OS Version: 14.4
 - Slackdump Version: 2.5.8

**Comment by rusq** [comment]
Funny one, thanks for the report, will fix it.

**Comment by rusq** [comment]
Fixed, will release in v2.5.9
